src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -193,17 +171,14 @@
     short-summary: Updates the display name of the management group. If null, no change is made.
   - name: --parent -p
     type: string
-    short-summary: Update the parent of the management group. Can be the fully qualified id or the name of the management group. If null, no change is made.
+    short-summary: Updates the parent of the management group. Can be the fully qualified id or the name of the management group. If null, no change is made.
```

Comment:
We need to use the first-person voice in the help message.
```suggestion
    short-summary: Update the parent of the management group. Can be the fully qualified id or the name of the management group. If null, no change is made.
```